### ISSUE TITLE
ipympl not working in scipy-notebook

### ISSUE THREAD

**Issue by mpmdean** [jupyter/docker-stacks#1038]
There appears to be a problem with ipympl not working within jupyterlab when using scipy-notebook.

1. Launch docker run -p 8888:8888 jupyter/scipy-notebook
2. Switch to jupyterlab mode by prepending lab to the URL.
3. Run notebook commands as:
![76247554-1a674700-6216-11ea-8a01-7d268ebb7746 (1)](https://user-images.githubusercontent.com/699991/76316787-821ea000-62b1-11ea-882a-57df897545cd.png)

**What do you expect to happen?**
Plot to appear 

**What actually happens?**
No plot. Error displaying widget: model not found

**Comment by romainx** [comment]
Hey just submitted a PR #1040 to fix it.

I let you check to confirm it's OK on your side.
Thanks
Best

**Comment by parente** [comment]
Fixed by #1039. 🤞

**Comment by romainx** [comment]
Good job, just pulled the latest version of the `scipy-notebook` and it works as expected with the following versions. Good job 👍 

```bash
jovyan@29e533608d50:~$ jupyter-labextension list
JupyterLab v2.0.1
Known labextensions:
   app dir: /opt/conda/share/jupyter/lab
        @bokeh/jupyter_bokeh v2.0.0  enabled  OK
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        jupyter-matplotlib v0.7.2  enabled  OK
```
